### ISSUE TITLE
Improve accessibility on Onomasticon module

### ISSUE THREAD

**Issue by 4hardfire** [openfed/openfed#37]
Currently, the "Onomaticon" module is using the element in a wrong implementation way.

The element represents the defining instance of a term and it can contain only the term being defined. If the ´title´ attribute of the element is present, then it must contain the term being defined only as well.

Defining term using a tooltip: By default, the paragraph, description list group, or section that is the nearest ancestor of the element must contain the definition(s) for the term given by the element. However, if no ancestor node, the ´aria-describedby´ property may be used to attach the definition(s) for the term to the element through the use of an ´id´ reference.

What is outputted currently but wrongly:

```
<dfn class="onomasticon ..." tabindex="0" title="marketplaces">
  <button disabled="" class=...">
    <p>Here the definition about marketplaces</p>
  </button>
</dfn>
```

What should be outputted according to the specifications and in compliance with the WCAG guidelines:

```
<span class="onomasticon ...">
  <dfn tabindex="0" aria-describedby="definition">marketplaces</dfn>
  <span id="definition" role="tooltip" hidden="">Here the definition about marketplaces</span>
</span>
```

To fix this issue we need to include the following patch from the following issue:

- Issue: https://www.drupal.org/project/onomasticon/issues/3340245
- Patch: https://www.drupal.org/files/issues/2023-02-08/onomasticon-enhance-accessibility-3340245-3.patch
